sophia/src/term.rs

```diff
@@ -206,50 +209,62 @@ where
         }
     }
 
-    /// Copy another term with the given factory,
-    /// applying the given normalization policy.
-    pub fn normalized_with<U, F>(other: &'_ Term<U>, mut factory: F, norm: Normalization) -> Term<T>
+    /// Transforms the underlying IRIs according to the given policy.
+    ///
+    /// If the policy already applies the Term is returned unchanged.
+    pub fn normalized(&self, policy: Normalization) -> Cow<'_, Self>
```

Comment:
I'm not a big fan of this change of API. Maybe my intent would have been clearer if I had named the method `copy_normalized_with`. The main goal was to *copy* the term, while ensuring a given "internal form" for IRIs...

This change means that `TermFactory::copy_normalized` has been commented out, which I think is a shame...

Reply:
Okay, I forgot to refactor that method in `TermFactory` :+1: .

Furthermore, I see now the use-case for `copy_normalized_with()`. I have to admit, I had a different use-case in mind. But your explanation makes sense. I'll remove this change.

Reply:
checkout changes in 488aaa8